OloEngine/tests/Rendering/PropertyTests/RendererAttachedTest.cpp

```
@@ -118,8 +120,23 @@ namespace OloEngine::Tests
+                    m_Scene->OnUpdateRuntime(ts);
+                }
+                // Drain the GL error queue this render tick produced. The full
+                // pipeline is known to emit benign stray GL errors (e.g.
+                // GL_INVALID_OPERATION binding a texture whose handle went stale
+                // across cross-test render-graph/asset churn — correct pixels,
+                // dirty error queue; tracked separately). Draining here extends
+                // the GLStateGuard "a render leaves no global GL state behind"
+                // contract to the error queue — the same containment the #485
+                // production fix applied in the readback helpers (69aa9357) — so
+                // a render in this fixture cannot poison the shared context of a
+                // later GPU test. The process-wide #485 listener still guards
```

Cppcheck (2.21.0)

[style] The function 'GetScene' is never used.

(unusedFunction)